get_info.php

```diff
@@ -1 +1,171 @@
+    $conn = new PDO(
+        "mysql:host=$servername;dbname=$database",
+        $username,
+        $password
+    );
+    // set the PDO error mode to exception
+    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    $stmt = $conn->prepare($query);
+    $stmt->execute();
+
+    if($method == "GET"){
+        //fetch all results
+        $fetched = $stmt->fetchAll(PDO::FETCH_ASSOC);
+        $result['TYPE'] = $_GET["what"] . "_package";
+        $result['DATA'] = $fetched;
+    }else{
+		$result = HTTPStatus(202);
+    }
+
+    echo json_encode($result);
+} catch (PDOException $e) {
+    $result = HTTPStatus(500);
+    date_default_timezone_set('America/Toronto');
+    error_log(date("Y-m-d h:i:sa")." : ".$e->getMessage()."\n", 3, "errors.log");
+    print json_encode($result, JSON_PRETTY_PRINT);
+    exit();
+}
+
+function handle_post($data){
+	error_check($data, TRUE);
+
+	$result = url_to_params(parse($data["url"]));
+
+	$keys = "";
+	$values = "";
+	foreach($data['data'] as $key=>$value){
+		$keys.=$key . ", ";
+		$values.="'". $value . "', ";
+	}
+	$keys = rtrim($keys, ", ");
+	$values = rtrim($values, ", ");
+
+	// Whitelist check
+	check_columns(array_keys($data['data']), $result['table'], 'post');
+
+	return "INSERT INTO ta_feedback.$result[table] ($keys) VALUES ($values);";
+}
+
+function handle_get(){
+    //error_check($_GET, FALSE);
+    switch($_GET["what"]){
+        case "tas":
+            return get_course_associations(TRUE);
+            break;
+        case "questions":
+            return get_questions();
+            break;
+        case "course_pairings":
+            return get_course_associations(FALSE);
+            break;
+        case "user_info":
+            return get_user_info();
+            break;
+
+        default:
+            throw new Exception("Invalid Page");
+            break;
+    }
+}
+
+/**
+* @return returns an SQL statment that updates a row
+*/
+function handle_put($data){
+    error_check($data, TRUE);
+
+    $url = parse($data["url"]);
+    $result = url_to_params($url);
+
+    // Whitelist check
+    check_columns(array_keys($data['data']), $result['table'], 'put');
+
+    $table = $result['table'];
+    $request_data = $data['data'];
+    $condition = $result['condition'];
+
+    $column = "";
+    foreach ($request_data as $key => $value) {
+        $column.= "$key = '$value',";
+    }
+    $column = rtrim($column, ", ");
+
+    return "UPDATE ta_feedback.$table SET $column WHERE $condition;";
+}
+
+/**
+* @return returns an SQL statment that deletes a row
+*/
+function handle_delete($data){
+    error_check($data, FALSE);
+
+    $result = url_to_params(parse($data["url"]));
+
+    // Whitelist check
+    check_columns(array_keys($data['data']), $result['table'], 'delete');
+
+    $table = $result['table'];
+    $condition = $result['condition'];
+
+    return "DELETE FROM ta_feedback.$table WHERE $condition;";
+}
+
+function HTTPStatus($num) {
```

Comment:
I don't like this name. Can you think of something more descriptive? This function returns an error object corresponding to an http status... `http_response_code_to_error` is a bit wordy, but it tells you what the function is doing.